cache/filter.go

```diff
@@ -55,6 +55,48 @@ func LabelFilter(label string) Filter {
 	}
 }
 
+// ActorFilter return a Filter that match a bug actor
+func ActorFilter(query string) Filter {
+	return func(repoCache *RepoCache, excerpt *BugExcerpt) bool {
+		query = strings.ToLower(query)
+
+		for _, id := range excerpt.Actors {
+			identityExcerpt, ok := repoCache.identitiesExcerpts[id]
+			if !ok {
+				panic("missing identity in the cache")
+			}
+
+			if strings.Contains(strings.ToLower(identityExcerpt.Name), query) ||
+				query == identityExcerpt.Id || query == strings.ToLower(identityExcerpt.Login) {
+				return true
+			}
+		}
+
+		return false
+	}
+}
+
+// ParticipantFilter return a Filter that match a bug participant
+func ParticipantFilter(query string) Filter {
+	return func(repoCache *RepoCache, excerpt *BugExcerpt) bool {
+		query = strings.ToLower(query)
+
+		for _, id := range excerpt.Participants {
+			identityExcerpt, ok := repoCache.identitiesExcerpts[id]
+			if !ok {
+				panic("missing identity in the cache")
+			}
+
+			if strings.Contains(strings.ToLower(identityExcerpt.Name), query) ||
+				query == identityExcerpt.Id || query == strings.ToLower(identityExcerpt.Login) {
+				return true
+			}
+		}
+
```

Comment:
You could use `gofmt` to reduce unnecessary white spaces :P I mean its not a necessacity but a good coding practice :/

Reply:
it is `gofmt`-ed. Sometimes it's good to leave space to simplify reading. But i agree last space before return isn't necessary